src/server/util/lrucache.ts

```diff
@@ -0,0 +1,98 @@
+import _ from 'lodash'
+
+const MAX_SIZE_BYTES = 2000
+
+interface CookieData {
```

Comment:
let's strive to make this LRUCache class standalone as an independent data structure - it currently violates the _Single Responsibility Principle_ because it knows a lot about the format of the cookies, mapping epoch to short links etc.

we could also use _types_ such as [Record<K, T>](https://www.typescriptlang.org/docs/handbook/utility-types.html#recordkt) instead of interfaces here.